osf_tests/test_preprint_summary.py

```diff
@@ -19,11 +18,33 @@ def preprint_provider():
 def preprint(preprint_provider):
     return PreprintFactory._build(PreprintService, provider=preprint_provider)
 
+
+@pytest.fixture()
+def yesterday_at_midnight():
+    return datetime.datetime.utcnow().replace(hour=0,
```

Comment:
If you have to use `...utcnow()`, you're probably better off using the timezone aware django util:
```
from django.utils import timezone
timezone.now()

```
See https://docs.djangoproject.com/en/1.11/topics/i18n/timezones/